corehq/apps/app_manager/static/app_manager/json/vellum-app-callout-templates.yaml

```diff
@@ -26,3 +26,7 @@
    icon: icon-vellum-android-intent
    name: Breath Counter
    id: org.commcare.respiratory.BREATHCOUNT
+-
+   icon: fa-id-badge
+   name: Simprints Enrolment
```

Comment:
nit, but in American English this is `Enrollment`. I don't know if we prefer American English, but we have no instances of the word `enrolment` anywhere in our codebase, and 13 instances of the word `enrollment`.

Reply:
In response to: https://github.com/dimagi/commcare-hq/pull/23476#discussion_r261694615 (since I think that comment was a response to this one)

That (`enrol`) is the correct spelling in non-American ("British") English. While there are lots of arguments that given our userbase we shouldn't be using American English, I think it's better that we remain consistent by sticking to using American English. (There's also inconsistency within this PR too: https://github.com/dimagi/commcare-hq/pull/23476/files#diff-7b62556ea96bb44d96bc6fff95cfa2f2R1061)

This is all very pedantic of me, but it's the kind of thing that can get confusing later.